pkg/services/serviceaccounts/manager/roles.go

```diff
@@ -8,9 +8,10 @@ import (
 var (
 	role = accesscontrol.RoleRegistration{
 		Role: accesscontrol.RoleDTO{
-			Version:     2,
+			Version:     3,
 			Name:        "fixed:serviceaccounts:writer",
-			Description: "",
+			DisplayName: "Service accounts writer",
+			Description: "Create, update, delete, read, or query service accounts.",
```

Comment:
Looks like it currently only contains `serviceaccounts.ActionDelete` permission. Should it also have permissions to create, read and edit?